services/api-server/src/simcore_service_api_server/services/webserver.py

```diff
@@ -228,24 +244,54 @@ async def create_project(self, project: ProjectCreateNew) -> ProjectGet:
             before_sleep=before_sleep_log(_logger, logging.INFO),
         ):
             with attempt:
-                data = await self.get(status_url)
-                task_status = TaskStatus.parse_obj(data)
+                status_data = await self.get(status_url)
```

Comment:
stupid question: did you try with several instances of the webserver?
I'm suddenly wondering, cause if the api-server talks "internally" to the webserver that means there is no stickyness right?

Reply:
not a stupid question at all. there is indeed no stickyness ... i have to indeed check the consequences of this.